tests/pkcli/roles_test.py

```diff
@@ -29,7 +30,8 @@ def _check_file(exists=True):
             [x.basename for x in pkio.walk_tree(fc.sr_user_dir(), _proprietary_file)],
         )
 
-    _proprietary_file = 'CapLaser.zip'
+    pkunit.data_dir().join('db').copy(sirepo.server._app.sirepo_db_dir)
```

Comment:
Why is this copy necessary?

Reply:
There is a dummy flash.rpm in roles_work/db/proprietary_code/flash/flash.rpm. Without it no file is copied to the lib dir and the test fails.

Reply:
Thanks. I think it would help me to document this. Why can't it use srdb.root()?

Reply:
Sure, I'll add a comment.

I was just copying what [email2_test does](https://github.com/radiasoft/sirepo/blob/7c33eba544cae022623b6618cf43d9226efbf005/tests/auth/email2_test.py#L28). From the looks of it I think we can get rid of all instances of _app.sirepo_db_dir and use srdb.root(). It is just [srdb.root() anyways](https://github.com/radiasoft/sirepo/blob/7c33eba544cae022623b6618cf43d9226efbf005/sirepo/server.py#L607). Agree?